src/rt/sched/cown.h

```diff
@@ -1199,10 +1204,18 @@ namespace verona::rt
 
         // Reschedule the other cowns.
         for (size_t s = 0; s < (senders_count - 1); s++)
-          senders[s]->schedule();
+        {
+          if (senders[s])
+            senders[s]->schedule();
+        }
+
+        bool should_break = senders[senders_count - 1] == nullptr;
```

Comment:
I don't think `should_break` can be false now?  I think this is dead code from previous implementation.